src/db.rs

```diff
@@ -805,6 +818,102 @@ impl Db {
         self.inner.get_with_options(key, options).await
     }
 
+    /// Get multiple values from the database with default read options.
```

Comment:
Can you add a note to these docs (and the _with_options function, below, too) to alert users that `keys` need not be sorted?